src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -444,6 +447,9 @@
   <data name="ERR_ValueCantBeNull" xml:space="preserve">
     <value>Cannot convert null to '{0}' because it is a non-nullable value type</value>
   </data>
+  <data name="ERR_RefCantBeDefault" xml:space="preserve">
+    <value>Cannot convert default to '{0}' because it is a value or nullable type</value>
+  </data>
```

Comment:
`ERR_Ref...` is confusing since it refers to a value type. And why not support `default` for value types?